docker/Dockerfile_passenger

```diff
@@ -7,6 +7,8 @@ ENV PORT=$PORT_ARG
 # Enable the web server gem
 ENV PASSENGER_TEST=true
 
+RUN bundle update
+
 # Name and run the application
 RUN ./docker/app_name_generator.sh Passenger >> /tmp/app_name.txt
```

Comment:
These changes just make it easy to see the app names. Not in love with the way I did it, but it's helpful to have.